atlasdb-container-test-utils/src/main/resources/docker-compose-cassandra.yml

```diff
@@ -7,3 +7,10 @@ services:
       - MAX_HEAP_SIZE=$CASSANDRA_MAX_HEAP_SIZE
       - HEAP_NEWSIZE=$CASSANDRA_HEAP_NEWSIZE
       - LOCAL_JMX=no
+
+  cli:
```

Comment:
Why was this necessary?

Reply:
The nodetool command is run on a separate container by default - I maintained that convention, but we hadn't needed to do so with the single-node setup, so that meant adding this container. Would it be safe to run the commands in the `cassandra` container instead?

Reply:
I think we can run it on the cassandra container but wont push on it if its significant lift.